README.md

```diff
@@ -10,62 +10,76 @@ The idea is to construct a resource that is for the RSE community and curated by
+RSE Competencies Toolkit is a resource to support RSEs (Research Software Engineers) in tracking and managing their
+professional development. It is currently in the early stages of development and we welcome contributions and feedback.
 
-Note that, while we've focussed on RSE during the early stages of development, this tool can be used for any roles. We plan to extend this to other research roles and welcome contributions and collaboration from anyone working in research, broadly defined.
+Note that, while we've focussed on RSE during the early stages of development, this tool can be used for any roles. We
+plan to extend this to other research roles and welcome contributions and collaboration from anyone working in research,
+broadly defined.
 
 RSE Competencies Toolkit comprises:
 
-1. An RSE competency framework, outlining a structured set of skills that are useful when working as an RSE, with examples of how these skills can be demonstrated at different levels of experience. Not all RSEs will or need to have all skills at all levels.
+1. An RSE competency framework, outlining a structured set of skills that are useful when working as an RSE, with
+   examples of how these skills can be demonstrated at different levels of experience. Not all RSEs will or need to have
+   all skills at all levels.
 2. A curated set of training resources, linked to the skills and levels from the competency framework.
 3. A tool to visualise and compare different competency profiles.
 
 RSE Competencies Toolkit aims to support the following uses:
 
-1. Recording and visualising your competency profile as an individual RSE
-2. Comparing competency profiles across a group of RSEs (e.g. to show the commonalities and variety across RSEs doing the same role at the same level at the same organisation, or comparing across organisations)
-3. Find high-quality training resources to improve skills in a particular competency
-4. Define aspirational competency profiles, illustrate the gap to your current profile and highlight training resources that could help bridge that gap.
+1. Recording and visualising your competency profile as an individual RSE.
+2. Comparing competency profiles across a group of RSEs (e.g. to show the commonalities and variety across RSEs doing
+   the same role at the same level at the same organisation, or comparing across organisations).
+3. Find high-quality training resources to improve skills in a particular competency.
+4. Define aspirational competency profiles, illustrate the gap to your current profile and highlight training resources
+   that could help bridge that gap.
 
 ### Current Status
 
 - [x] Initial Research
-- [ ] Initial Development <-- We are Here  
+- [ ] Initial Development <-- We are Here
 - [ ] Minimum viable product
-- [ ] Alpha Release  
-- [ ] Feature-Complete Release  
+- [ ] Alpha Release
```

Comment:
Do we need a beta release before the alpha and the feature-complete as an intermediate step? Be more iterative?